src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -1,45 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  Icon,
-  IconButton,
-  OverlayTrigger,
-  Stack,
-  Tooltip,
+  Icon, IconButton, OverlayTrigger, Stack, Tooltip,
 } from '@edx/paragon';
 import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
 
 const AssignmentRowActionTableCell = ({ row }) => {
-  const cancelButtonMarginLeft = row.original.state === 'allocated' ? 'ml-2.5' : 'ml-auto';
+  const learnerStateBool = row.original.learnerState === 'waiting';
+  const cancelButtonMarginLeft = learnerStateBool ? 'ml-2.5' : 'ml-auto';
   return (
-    <div className="d-flex">
-      {row.original.state === 'allocated' && (
-        <>
-          <OverlayTrigger
-            key="Remind"
-            placement="top"
-            overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
-          >
-            <IconButton
-              className="ml-auto mr-0"
-              src={Mail}
-              iconAs={Icon}
-              alt="Remind learner"
+    <Stack direction="horizontal" gap="1">
+      {learnerStateBool && (
+      <OverlayTrigger
+        key="Remind"
+        placement="top"
+        overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
```

Comment:
[sanity check, non-blocking] Given the "Remind" icon button's tooltip in each row would the same `id` (i.e., `tooltip-remind`), I'm wondering if there could be a possibility that more than 1 `Tooltip` is rendered at the same time? The concern is that `id`'s in HTML need to be unique (can't have duplicate IDs on the page). Even if there is guaranteed to be at most one `Tooltip` rendered at a time, it may be a good idea to make the `id` for these tooltips unique per row.

Reply:
If it's guaranteed that multiple can never be rendered, I don't see the possibility of confusion? I could tack on the uuid on the end of it if you want to make sure though

Reply:
Yeah, I agree it's seemingly very low possibility to have more than 1 `Tooltip` rendered at once (the one instance that might be a factor that I can think of is that perhaps when quickly hovering over one `IconButton` to another in quick succession, due to the animations where one `Tooltip` fades in while the other `Tooltip` is fading out, where both could theoretically be rendered simultaneously.

Might be good to tack on the uuid to the end just in case.